conf/.env.docker

```diff
@@ -32,3 +32,5 @@ REDIS_PORT="{{REDIS_PORT}}"
 REDIS_PASSWORD="{{REDIS_PASSWORD}}"
 
 GITHUB_TOKEN="{{GITHUB_TOKEN}}"
+
+MAX_CHILDREN="{{MAX_CHILDREN}}"
```

Comment:
I also had the thaughts, but in a sense wouldnt this be part of PHP_ or FPM_?
Or keep it simple and go with NGINX?